trulens_eval/trulens_eval/tests/groundedness_smoke_tests.ipynb

```diff
@@ -13,24 +13,14 @@
     "SummEval is one of the datasets dedicated to automated evaluations on summarization tasks, which are closely related to the groundedness evaluation in RAG with the retrieved context (i.e. the source) and response (i.e. the summary). It contains human annotation of numerical score (**1** to **5**) comprised of scoring from 3 human expert annotators and 5 croweded-sourced annotators. There are 16 models being used for generation in total for 100 paragraphs in the test set, so there are a total of 16,000 machine-generated summaries. Each paragraph also has several human-written summaries for comparative analysis. \n",
```

Comment:
<p>as our <strong>expected_score</strong> and <s>u </s>to match</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/truera/trulens/pull/634/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>